### DISCUSSION TITLE
Community Website - Allow 3 Tiers of Drop-Down Tabs for Navigation

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#567] [Suggestions]
In the editor for the navigation menu, there appears to be a third tier slot for moving pages when organizing them, but attempting to settle a page in that "third slot" just bounces it back to its original slot. For the sake of organization, would it be possible to allow for community websites to support that third branch of sub-pages in the navigation? 
![image](https://github.com/gameserverapp/Platform/assets/44717221/01d15258-a05e-485c-ae77-459905c4c3a7)

**Comment by mvaessen** [comment]
Thanks for the suggestion, this is now available!
